vagrant/prepare-disk.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+# To start again and wipe out the disk, in this order:
+#
+#   wipefs -a /dev/sdb1
+#   fdisk /dev/sdb ; d; w
+
+DATA_MOUNT=/mnt/data
+
+mkdir -p $DATA_MOUNT
+
+if [ ! -b /dev/sdb1 ]; then
+    echo "Adding partition table to /dev/sdb"
+    echo ';' | sfdisk /dev/sdb
+else
+    echo "Partition table already exists"
+fi
+
+if [[ $(file -sL /dev/sdb1 | grep "/dev/sdb1: data$") ]]; then
```

Comment:
So we're looking to see if this partition is called 'data'? But how does it get called data?